rt/rs/sse/src/main/resources/META-INF/beans.xml

```diff
@@ -2,5 +2,5 @@
 <beans xmlns="http://java.sun.com/xml/ns/javaee"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
 	xsi:schemaLocation="http://java.sun.com/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/beans_1_1.xsd"
-	bean-discovery-mode="all">
```

Comment:
Agree than `all` was not correct here, but I would like SSE to be discoverable with no efforts when run in CDI container. I think we need only one single class to be discoverable , `SseFeature`, so @johnament could we add some exclusion / inclusion rules instead of just cutting everything off?

Reply:
Since `SseFeature` is a core CXF feature, not a JAX-RS feature, I believe its autoloaded that way.  Isn't it?

Reply:
It is core CXF feature, right, and is it automatically discovered by CDI extension (as an annotated type). But I believe if we set `bean-discovery-mode="none"`, it won't be automatically discovered  anymore, no?

Reply:
I'll test it out and get back to you.  If it doesn't work, I'd actually prefer to move it into the extension.

Reply:
I ended up going the route of adding the feature's behavior into the extension.  Thoughts?